services-directions/src/main/java/com/mapbox/api/directions/v5/MapboxDirections.java

```diff
@@ -184,14 +184,14 @@ private void errorDidOccur(@Nullable Callback<DirectionsResponse> callback,
     Converter<ResponseBody, DirectionsError> errorConverter =
       getRetrofit().responseBodyConverter(DirectionsError.class, new Annotation[0]);
     if (callback == null) {
-      BindingContextFactory.LOGGER.log(
+      Logger.getLogger(MapboxDirections.class.getName()).log(
```

Comment:
Instead of duplicating `Logger.getLogger(MapboxDirections.class.getName())` several times, could you extract it as a local private class-level variable instead?